dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOptionTest.java

```diff
@@ -81,4 +89,32 @@ public void testConfigureAdditionalMavenRepositories_addingSpringRepository()
     List<Path> paths = ClassPathBuilder.artifactsToClasspath(ImmutableList.of(artifact));
     Truth.assertThat(paths).isNotEmpty();
   }
+
+  @Test
+  public void testConfigureAdditionalMavenRepositories_invalidRepositoryUrl()
+      throws ParseException {
+
+    List<String> invalidUrls =
```

Comment:
I think it would be better to have a private method like `assertMavenRepositoryIsInvalid(String)`, and to have this test call it explicitly:

```
assertMavenRepositoryIsInvalid("foobar");
assertMavenRepositoryIsInvalid("_http_file__https");
// etc.
```

That way the failure would point to the exact line number that failed.